.devcontainer/setup.sh

```
@@ -0,0 +1,60 @@
+fi
+
+# If Doppler is already installed, skip everything
+if command -v doppler >/dev/null 2>&1; then
+	echo "Doppler already installed; skipping."
+else
+	sudo apt-get install -y doppler
+	echo "Doppler installed successfully."
+fi
+
+# install npm from nvm if it doesn't exist
+if [ ! command -v npm ] >/dev/null 2>&1; then
```

Shellcheck (0.11.0)

[warning] This expression is constant. Did you forget the $ on a variable?

(SC2050)